modules/models.py

```diff
@@ -3019,6 +3078,13 @@ class Lineup(BaseModel):
     elo_after_game_moonrise = SmallIntegerField(default=None, null=True)
     elo_after_game_global_moonrise = SmallIntegerField(default=None, null=True)
 
+    def as_json(self) -> dict[str, Any]:
```

Comment:
here as well

Reply:
This is a feature of Python 3.9, see [PEP 585](https://www.python.org/dev/peps/pep-0585/). If compatibility with 3.7 and 3.8 is required, `from __future__ import annotations` can be used.